### PR TITLE
Phases feature.

### PR DESCRIPTION
**Pull request by isht3**
- `evalai challenge CHALLENGE phases`
- `evalai challenge CHALLENGE phase PHASE`

Most of the content in the PR has been reviewed in the two open PR's right now.

The request functions have been reviewed in the closed PR. #19
The only part to be reviewed is the design of the CLI in `evalai/challenges.py`

### CODE REVIEW

File: evalai/utils/challenges.py [hunk 1]
```diff
@@ -160,3 +160,128 @@ def get_challenge_count(is_host=False, is_participant=False):
...
+                                      bold=True, fg="blue"))
+    phase_id = "Phase ID: {}\n\n".format(style(str(phase["id"]),
+                                         bold=True, fg="blue"))
+
+    title = "{} {} {}".format(phase_title, challenge_id, phase_id)
+
+    cleaned_desc = BeautifulSoup(phase["description"], "lxml").text
+    description = "{}\n".format(cleaned_desc)
+
+    start_date = "Start Date : " + style(phase["start_date"].split("T")[0],
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: Please use `.format` to concatenate the strings everywhere.

File: evalai/utils/challenges.py [hunk 2]
```diff
@@ -160,3 +160,128 @@ def get_challenge_count(is_host=False, is_participant=False):
...
+                                         fg="green")
+    start_date = "\n{}\n".format(style(start_date, bold=True))
+
+    end_date = "End Date : " + style(phase["end_date"].split("T")[0], fg="red")
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: Same as above.

File: evalai/utils/challenges.py [hunk 3]
```diff
@@ -160,3 +160,128 @@ def get_challenge_count(is_host=False, is_participant=False):
...
+    headers = get_headers()
+
+    try:
+        response = requests.get(url, headers=headers)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError as err:
+        echo(style("Error: " + response.json()['error'], fg="red", bold=True))
+        sys.exit(1)
+    except requests.exceptions.RequestException as err:
+        echo(err)
```
Comment (RishabhJain2018) [inline evalai/utils/challenges.py]: @varunagarwal @deshraj I think we should this error somewhere and display a message to user like `You are not connected to the internet. Please check your connection` What do you say?

**Comment by RishabhJain2018** [comment]
@isht3 Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
@isht3 Can you please resolve the conflicts here?

**Comment by isht3** [comment]
@RishabhJain2018 Done!

**Comment by RishabhJain2018** [comment]
@isht3 Please resolve the merge conflicts.

**Comment by RishabhJain2018** [comment]
@varunagrawal Please review this one too once @isht3 resolves the merge conflict.

**Comment by isht3** [comment]
@RishabhJain2018 Done!

**Pull request closed without merging**